src/DeploymentSlotSwapper.ts

```diff
@@ -34,7 +34,7 @@ export class DeploymentSlotSwapper extends WizardBase {
 class SwapStep extends WizardStep {
     private _subscription: SubscriptionModels.Subscription;
     private _sourceSlotNode: IAzureNode<DeploymentSlotTreeItem>;
-    private targetSlot: DeploymentSlotTreeItem | undefined;
+    private targetSlotNode: IAzureNode<DeploymentSlotTreeItem | undefined>;

```

Comment:
"Refactor deploymentSwapper to hold node as data rather than site to allow refreshing node after a swap"

^ Did you mean to call refresh somewhere in this file? I don't see it

Reply:
No, I haven't implemented that part yet, but I needed to refactor the data to hold the nodes for when I do implement the refresh File Node on slot swap.